src/collections/members/abhinay-jain/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+name: Abhijay Jain
+position: Contributor
+image_path: ./abhijay-jain.jpg
+github: Abhijay007
+twitter: CodeInVeins
+linkedin: abhijay-jain-551b01193
+location: India
```

Comment:
@Abhijay007 share your city name, if you would.

Reply:
Then, @ShivamTyagi12345 make it `City, Country`

Reply:
> Then, @ShivamTyagi12345 make it `City, Country`

`City : Delhi`